templates/_includes/head-polyfills.html

```diff
@@ -1,4 +1,4 @@
 <!-- ES5 Adapter -->
-<script src="https://ab2b55b0a26bb32fdc01-2c8b94bcec8e01c366189541a4035ea4.ssl.cf5.rackcdn.com/custom-elements-es5-adapter.js"></script>
+<script src="{{ site.url }}/node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
```

Comment:
This is an incorrect implementation of the ES5 adapter. 

Please move to the footer using the technique provided in the [vertical layout template](https://github.com/rackerlabs/helix-ui/blob/master/docs/components/layouts/vertical-layout-template.html#L106-L116).